### DISCUSSION TITLE
Duda sobre git

### DISCUSSION THREAD

**Discussion by gersondlarosa** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#488] [Q&A] [answered]
Buena tarde tengo una duda al tratar de crear mi repositorio de vue me arroja lo siguiente alguien me podria indicara que es lo que causa este error agrego error

error: remote origin already exists.

**Comment by Burstniart** [comment]
Que ya creaste otros repositorios asignandoles "origin" asi que los estas referenciando, puedes remover el remote origin o actualizar la url al repositorio de vue para solucionarlo

**Comment by doguedogue** [comment, marked as answer]
valida la configuración que ya tienes asi:
`git config --list`

y actualizalo como comenta @Burstniart

**Reply by gersondlarosa** [reply]
ya lo subió pero no me subió el nodmodule es esta bien ???

**Comment by gersondlarosa** [comment]
a ok lo voy a probar
